ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/electra/helpers/BeaconStateAccessorsElectra.java

```diff
@@ -46,7 +45,41 @@ public BeaconStateAccessorsElectra(
    */
   public UInt64 getValidatorMaxEffectiveBalance(final Validator validator) {
     return predicatesElectra.hasCompoundingWithdrawalCredential(validator)
-        ? maxEffectiveBalanceElectra
-        : minActivationBalance;
+        ? configElectra.getMaxEffectiveBalanceElectra()
+        : configElectra.getMinActivationBalance();
+  }
+
+  /**
+   * get_activation_exit_churn_limit
+   *
+   * @param state - the state to use to get the churn limit from
+   * @return Return the churn limit for the current epoch dedicated to activations and exits.
+   */
+  public UInt64 getActivationExitChurnLimit(final BeaconStateElectra state) {
+    return getChurnLimit(state).min(configElectra.getMaxPerEpochActivationExitChurnLimit());
+  }
+
+  /**
+   * get_churn_limit
+   *
+   * @param state the state to read active balance from
+   * @return Return the churn limit for the current epoch.
+   */
+  public UInt64 getChurnLimit(final BeaconStateElectra state) {
+    final UInt64 churn =
+        configElectra
+            .getMinPerEpochChurnLimitElectra()
+            .max(getTotalActiveBalance(state).dividedBy(configElectra.getChurnLimitQuotient()));
+    return churn.minusMinZero(churn.dividedBy(configElectra.getEffectiveBalanceIncrement()));
```

Comment:
Is this division correct? The spec has it as the `churn mod EFFECTIVE_BALANCE_INCREMENT`

From spec:
```
return churn - churn % EFFECTIVE_BALANCE_INCREMENT
```